app/Console/Commands/NotifyVerificationCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Mail\EmailConfirm;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+
+class NotifyVerificationCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'users:notify-verification';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send notification to unverified users';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $users = User::all();
```

Comment:
так как пользователей может быть много в базе
лучше выбрать сразу тех у кого email_verified_at = null
```suggestion
        $users = User::query()->whereNull('email_verified_at')-get();
```